.github/workflows/Manager-CI.yaml

```diff
@@ -20,7 +20,9 @@ jobs:
       FLEET_MANAGER_CONTAINER_NAME: openbridge/fleet-manager:${{ github.sha }}
       FLEET_SHARD_MINIKUBE_CONTAINER_NAME: openbridge/fleet-shard:${{ github.sha }}
       EXECUTOR_CONTAINER_NAME: openbridge/executor:${{ github.sha }}
-      INGRESS_CONTAINER_NAME: openbridge/ingress:${{ github.sha }}
+      INGRESS_CONTAINER_NAME: openbridge/ingress:${{ github.sha }}# KIND_MAIN_NODE_HOSTNAME is used for SendToBridge action, so that we can access the bridge ingress from a processor pod
```

Comment:
```suggestion
      INGRESS_CONTAINER_NAME: openbridge/ingress:${{ github.sha }}
      # KIND_MAIN_NODE_HOSTNAME is used for SendToBridge action, so that we can access the bridge ingress from a processor pod
```